contrib/osx/README.md

```diff
@@ -7,7 +7,7 @@ This guide explains how to build Electron Cash binaries for macOS systems.
 We build our binaries on El Capitan (10.11.6) as building it on High Sierra
 makes the binaries incompatible with older versions.
 
-This assumes that the Xcode + Xcode Command Line tools (and thus git) are already installed. You can install older (and newer!) versions of Xcode from Apple provided you have a devloper account [from the Apple developer downloads site](https://developer.apple.com/download/more/).
+This assumes that the Xcode Command Line tools (and thus git) are already installed. You can install older (and newer!) versions of Xcode from Apple provided you have a devloper account [from the Apple developer downloads site](https://developer.apple.com/download/more/).
```

Comment:
You cannot install Xcode Command Line tools without installing Xcode.  I am unsure for the reason behind this edit....?

Reply:
You can. I have the command line tools with Xcode (which is like 10GB big).

To do this, simply run the command `xcode-select --install`.

[Here is a tutorial](https://www.maketecheasier.com/install-command-line-tools-without-xcode/).

Reply:
This I did not know.  Thanks.  TIL.  Resolving.   Thanks!